packages/cactus-plugin-keychain-aws-sm/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@hyperledger/cactus-plugin-keychain-aws-sm",
+  "version": "0.4.1",
+  "description": "A keychain implementation storing its entries in AWS Secret Manger.",
+  "main": "dist/lib/main/typescript/index.js",
+  "mainMinified": "dist/cactus-plugin-keychain-aws-sm.node.umd.min.js",
+  "browser": "dist/cactus-plugin-keychain-aws-sm.web.umd.js",
+  "browserMinified": "dist/cactus-plugin-keychain-aws-sm.web.umd.min.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/types/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "generate-sdk": "openapi-generator generate --input-spec src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/",
```

Comment:
@jagpreetsinghsasan Depending on which PR gets merged first please watch out for having to update this and also to regenerate the API client sources (the other PR being the one where Travis upgrades the generator to the latest and greatest for the whole project)